releng/org.eclipse.xtext.runtime.feature/feature.xml

```diff
@@ -29,7 +29,7 @@ http://www.eclipse.org/legal/epl-v10.html Description here.
       <import plugin="com.google.guava" version="14.0.0" match="greaterOrEqual"/>
       <import plugin="javax.inject" version="1.0.0" match="greaterOrEqual"/>
       <import plugin="com.google.inject" version="3.0.0" match="greaterOrEqual"/>
-      <import plugin="org.objectweb.asm" version="6.1.1" match="compatible"/>
+      <import plugin="org.objectweb.asm" version="6.2.0" match="equivalent"/>
```

Comment:
Shouldn’t this be 6.2.1?

Reply:
this is not available yet. i hate eclipse :(